src/main/java/seedu/duke/commands/DoneCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.duke.commands;
+
+import seedu.duke.trackables.Task;
+
+import java.util.List;
+
+public class DoneCommand extends Command {
+
+    private int taskId;
+
+    public DoneCommand(int taskId) {
+        this.taskId = taskId - 1;
+    }
+
+    @Override
+    public void execute(List<Task> tasks) {
+        tasks.get(taskId).markAsDone();
+        echo(
```

Comment:
The indentation for wrapped lines in execute method should be 8 spaces.

Reply:
Line 19 should be together with Line 18.